docs/streams/index.html

```diff
@@ -201,7 +201,23 @@ <h3 style="max-width: 75rem;">The easiest way to write mission-critical real-tim
             </pre>
         </div>
     </div>
-
+     <h3 class="customer-title">See how Kafka Streams is being used</h3>
+    <div class="customer__cards">
+        <div class="customer__card">
+          <div class="customer__card__icon">
+            <img src="/0110/images/icons/rabobank.png">
```

Comment:
I think we want these to be relative paths, otherwise it's going to be a pain to maintain as we update to new versions. Or I guess just use `{{version}}` in place of `0110` as seems to be done earlier in this file.